### ISSUE TITLE
Are software agents legitimate values for xBy terms?

### ISSUE THREAD

**Issue by baskaufs** [tdwg/dwc#318]
This question was brought up by @danstowell for the case where a media item was automatically classified to a taxon by an algorithm. In such a case it might be legitimate to use an identifier for an algorithm or software agent as a value for `dwc:identifiedBy`/`dwciri:identifiedBy`. However, the [term definition](https://dwc.tdwg.org/list/#dwc_identifiedBy) only lists people or groups of people as possible values.  Has this term been used where the value is a non-human agent? Should it be?

This question could apply to any of the other xBy terms (recordedBy, georeferencedBy). If non-human agents are acceptable values, this issue could morph into a term change proposal.

**Comment by matdillen** [comment]
This topic has also been discussed in the Attribution Interest group: tdwg/attribution#38

A quick look into GBIF found a few occurrence records with `iNaturalist` as value for identifiedBy and numerous variations of `camera` for recordedBy. 

A key question is: how do we recommend software (or hardware) agents should be identified?

**Comment by danstowell** [comment]
I agree. Re the key question, I made a comment on the other thread you referenced: https://github.com/tdwg/attribution/issues/38#issuecomment-781242715

**Comment by tucotuco** [comment]
I think this would make a positive change for all of the XBy terms. It satisfies all three of the justification requirements (demand, efficacy, and stability) set forth in the Vocabulary Maintenance Specification Section 3.1 Justifications for change. I recommend that term change requests be created for each of the affected terms using the Change term template (Create a new issue, choose the Change Term template, fill in everything in the template.

**Comment by tucotuco** [comment]
For those still interested in these changes, please provide [Term change requests](https://github.com/tdwg/dwc/issues/new?assignees=&labels=Term+-+change&projects=&template=term-change-template.md&title=Change+term+-+).

**Comment by baskaufs** [comment]
Tagging as "appears to have consensus", assuming that dcterms:Agent will get worked out.

**Comment by tucotuco** [comment]
Answered and addressed by the adoption of dcterms:Agent in Darwin Core.
